rllib/examples/offline_rl/pretrain_bc_single_agent_evaluate_as_multi_agent.py

```diff
@@ -0,0 +1,167 @@
+# @HybridAPIStack
+
+"""Example showing how to train a (SA) BC RLModule while evaluating in a MA setup.
+
+Here, SA=single-agent and MA=multi-agent.
+
+Note that the BC Algorithm - by default - runs on the hybrid API stack, using RLModules,
+but not EnvRunners or ConnectorV2s yet.
```

Comment:
Probably adding here that is specifically doe snot use `SingleAgentEpisode/MultiAgentEpisode`?